test/test_trio_server/test_full_sync.py

```diff
@@ -0,0 +1,47 @@
+import unittest
+from test.test_trio_server.mock_lean import \
+    LeanShouldGetRequest, LeanShouldNotGetRequest, LeanSendsResponse, start_with_mock_lean
+from lean_client.trio_server import TrioLeanServer
+import trio
+import trio.testing
+
+
+class TestFullSync(unittest.TestCase):
```

Comment:
Oh! I strongly suggest we don't go back 15 years in time, and use pytest instead. Switching is very easy here. Just remove that line, and deindent the next function. Then remove the last two lines of this file. Sometimes progress is simply removing cruft.

Reply:
Sure.  I'm not so very familiar with different python testing frameworks.  I can use pytest.